app/controllers/clinicfinders_controller.rb

```diff
@@ -5,10 +5,10 @@ def search
     return head :bad_request if params[:zip].blank?
 
     clinic_finder = ClinicFinder::Locator.new(
-      Clinic.where(:zip.nin => [nil, '']),
+      Clinic.where(:zip.nin => [nil, '', '99999']),
```

Comment:
doing this because this isn't a real zip code and gives us an out to have clinics not return. (This is the zip code of DCAF's catchall 'OTHER' option.)

Reply:
Should we comment that in the code for future us's confusion alleviation?